packages/gatsby-source-graphql/package.json

```diff
@@ -7,12 +7,11 @@
     "url": "https://github.com/gatsbyjs/gatsby/issues"
   },
   "dependencies": {
+    "@apollo/client": "^3.5.8",
     "@babel/runtime": "^7.15.4",
-    "@graphql-tools/links": "^7.1.0",
-    "@graphql-tools/utils": "^7.10.0",
-    "@graphql-tools/wrap": "^7.0.8",
-    "apollo-link": "1.2.14",
-    "apollo-link-http": "^1.5.17",
+    "@graphql-tools/links": "^8.2.1",
+    "@graphql-tools/utils": "^8.6.1",
+    "@graphql-tools/wrap": "^8.3.3",
```

Comment:
since these `@graphql-tools` depend on each other, figured it made sense to bump all of them here